de.mpicbg.knime.knutils/build.properties

```diff
@@ -1,5 +1,6 @@
-bin.includes = knutils.jar,\
-               META-INF/
+bin.includes = knutils.jar
```

Comment:
@gab1one
is it necessary to have this entry twice in that file?

Reply:
Good catch we only need it once